darwin/exporter/formats/semantic_mask.py

```diff
@@ -0,0 +1,121 @@
+from pathlib import Path
+from typing import Generator, List, Optional
+
+import numpy as np
+from PIL import Image
+from tqdm import tqdm
+from upolygon import draw_polygon
+
+import darwin.datatypes as dt
+
+
+def export(annotation_files: Generator[dt.AnnotationFile, None, None], output_dir: Path):
+    masks_dir = output_dir / "masks"
+    masks_dir.mkdir(exist_ok=True, parents=True)
+    annotation_files = list(annotation_files)
+    categories = calculate_categories(annotation_files)
+    ignore_idx = 255
+    pbar = tqdm(annotation_files)
+    pbar.set_description(desc="Processing annotations", refresh=True)
+    for annotation_file in pbar:
+        outfile = masks_dir / (annotation_file.path.stem + ".png")
```

Comment:
I always prefer string interpolation over concatenation, but not too important
```suggestion
        outfile = masks_dir / f'{annotation_file.path.stem}.png')
```